test/restart/pytest.ini

```diff
@@ -0,0 +1,5 @@
+# Pytest configuration file. If we don't have one in this directory,
+# pytest will look for one in our ancestor directories, and may find
+# something irrelevant. So we should have one here, even if empty.
```

Comment:
This comment is no longer relevant (the file isn't empty).
So if you keep this suite (I hope we don't need to), let's drop this comment.

Reply:
I dropped this suite, so there is no problem with this either